src/sugar.rs

```diff
@@ -111,6 +96,20 @@ macro_rules! proptest {
         $($(#[$meta])*
           fn $test_name($($parm in $strategy),+) $body)*
     } };
+
+    (|($($parm:pat in $strategy:expr),+)| $body:block) => {
+        || {
+            let config = $crate::test_runner::Config::default();
```

Comment:
We should check that the config doesn't have `fork()` enabled since that feature requires proptest having control over the top-level test function.

Reply:
Thanks!

Ended up checking for both `fork` and `timeout` features, even though one depends on another. Possibly worth to introduce a `no_fork` flag in config which takes precedence to make this clearer.